packages/cli/src/commands/multi-chain/deploy.ts

```diff
@@ -71,7 +72,8 @@ export default class MultiChainDeploy extends Command {
     flags.projectName = await valueOrPrompt(flags.projectName, 'Enter project name', 'Project name is required');
 
     // Multichain query descriptor
-    const ipfsCID = fileToCidMap.get(path.basename(multichainManifestPath));
+    const ipfsCID = Array.from(fileToCidMap).find(([file]) => file === '')?.[1];
+    assert(ipfsCID, 'Multichain deployment CID not found');
```

Comment:
I think this error could be clearer, it doesn't give the user any information how to resolve the issue.